.githooks/pre-commit

```diff
@@ -45,6 +45,18 @@ else
     against=16bbb57
 fi
 
+[[ -z "$files" ]] && exit
+# Change the copyright date at the top of any text files
```

Comment:
Does this apply to all files or just the ones that were changed in the commit?

Reply:
Should be all of them. I grabbed this from RocmBLAS commit Umang showed me.

Reply:
We should reuse the query from files from below at line 61 and then assign it to the `files` variable: `files=$(git diff-index --cached --name-only $against | grep -E '\.h$|\.hpp$|\.cpp$|\.cl$|\.h\.in$|\.hpp\.in$|\.cpp\.in$|\.py$')`, and then use the `$files` variable below instead.